ajax/dropdownAuthorization.php

```diff
@@ -38,11 +38,9 @@
 global $DB;
 
 $iterator = $DB->request(
+    PluginOauthimapAuthorization::getTable(),
     [
-        'FROM'  => PluginOauthimapAuthorization::getTable(),
-        'WHERE' => [
-            PluginOauthimapApplication::getForeignKeyField() => $_POST['application_id'] ?? null,
-        ],
+        PluginOauthimapApplication::getForeignKeyField() => $_POST['application_id'] ?? null,
     ],
 );
```

Comment:
To revert, the original version is already GLPI 11 compatible, whereas your proposal is not.